devbin/setup-vm-with-clone-db-access.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex
+
+apt-get update
+apt-get install -y emacs-nox git mysql-client python3-pip
+
+mkdir /sql-config/
+chmod 777 /sql-config/
+
+git clone https://github.com/hail-is/hail.git
+cd hail
+
+pip3 install -r hail/python/requirements.txt
+pip3 install -r hail/python/dev/requirements.txt
+pip3 install -r hail/python/hailtop/requirements.txt
+pip3 install -r gear/requirements.txt
```

Comment:
We try to use `pinned-requirements.txt` files as much as possible for reproducibility. You can instead do

```
pip install \
    -r hail/python/pinned-requirements.txt \
    -r hail/python/dev/pinned-requirements.txt \
    -r gear/pinned-requirements.txt
```